packages/suite/src/utils/wallet/coinmarket/coinmarketUtils.ts

```diff
@@ -172,3 +173,16 @@ export const getTagAndInfoNote = (quote: { infoNote?: string }) => {
 
     return { tag, infoNote };
 };
+
+export const getDefaultCountry = (country: string = regional.unknownCountry) => {
+    const label = regional.countriesMap.get(country);
+
+    if (label === undefined) {
+        throw new Error(`Unknown country: ${country}`);
```

Comment:
This still crashes the app, at least in develop. Will you ask Invity or would you like me to do it? :)

Reply:
Apparently it's some unclear and rare thing. resolving, but should later be discussed with martin from invity